### DISCUSSION TITLE
Variable naming

### DISCUSSION THREAD

**Discussion by tianmeioop** [yalmip/YALMIP#1386] [Problems using YALMIP] [answered]
I often use it for power dispatch modeling. However, it is impossible to name the variables so that you can use the LP file to verify that the modeling is correct, or to see if there are any missing constraints and variables.

**Comment by johanlofberg** [comment, marked as answer]
I guess you are talking about exporting a model to a .lp file?

The support for exporting models to .lp, or .mod etc is limited and a very unprioritized part of YALMIP. It would not trivial to do this, impossible in many cases, as YALMIP has no notion of variable names and does not know what you have called your variable. To YALMIP, a variable is only an index, but then when you write the code you have placeholders for these that you give names.

**Comment by tianmeioop** [comment]
Thanks for your answer！
&nbsp;


We&nbsp;are&nbsp;infinite.
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Johan ***@***.***&gt;; 
发送时间: 2024年3月21日(星期四) 下午4:23
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [yalmip/YALMIP] Variable naming (Discussion #1386)





 
I guess you are talking about exporting a model to a .lp file?
我猜您正在谈论将模型导出到 .lp 文件？
 
The support for exporting models to .lp, or .mod etc is limited and a very unprioritized part of YALMIP. It would not trivial to do this, impossible in many cases, as YALMIP has no notion of variable names and does not know what you have called your variable. To YALMIP, a variable is only an index, but then when you write the code you have placeholders for these that you give names.
对将模型导出到 .lp 或 .mod 等的支持是有限的，并且是 YALMIP 中非常不优先的部分。这样做并不简单，在许多情况下是不可能的，因为 YALMIP 没有变量名的概念，也不知道您如何称呼变量。对于 YALMIP 来说，变量只是一个索引，但是当您编写代码时，您会为您指定的变量提供占位符。
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
直接回复此电子邮件、在 GitHub 上查看或取消订阅。
You are receiving this because you authored the thread.
您收到此消息是因为您创作了该主题。  Message ID: ***@***.***&gt;
